src/variables-tree.js

```diff
@@ -7,8 +8,19 @@ export function buildVariablesTree(variables, query) {
     if (!query) {
       return true
     }
-    const normalizedQuery = query.toLowerCase()
-    return variable.name.toLowerCase().includes(normalizedQuery)
+    function nameMatches(variable, query) {
+      const normalizedName = variable.name.toLowerCase()
+      return normalizedName.includes(query)
+    }
+    function labelMatches(variable, query) {
+      if (!variable.label) {
+        return false
+      }
+      const normalizedLabel = removeAccents(variable.label.toLowerCase())
```

Comment:
I first tried search-as-you-type and yes, it was slow.

We could store the accent-folded tree, but as it was slow just with name search #35, it seems that it would be premature optimization. The accent folding is not the only responsible.

We could debounce the search-as-you-type too, but let's keep it with an action for now.